runtimes/runtimes/webworker.ts

```diff
@@ -128,18 +126,10 @@ export const webworker = (props: RuntimeProps) => {
         // Set up LSP events handlers per server
         const lsp: Lsp = {
             addInitializer: lspServer.setInitializeHandler,
-            onInitialized: handler =>
-                lspConnection.onInitialized(p => {
-                    const workspaceCapabilities = lspRouter.clientInitializeParams?.capabilities.workspace
-                    if (workspaceCapabilities?.didChangeConfiguration?.dynamicRegistration) {
-                        // Ask the client to notify the server on configuration changes
-                        lspConnection.client.register(DidChangeConfigurationNotification.type, undefined)
-                    }
-                    handler(p)
-                }),
+            onInitialized: lspServer.setInitializedHandler,
```

Comment:
Why allowing Server to set `InitializedHandler` is needed? 
How this will impact server implementations, will every server now need to provide this handler with logic removed above?

Reply:
Please ignore, I noticed that this logic is moved to LSPRouter, which runs it before calling handler the old way. So this is just refactoring and does not change current implementation, right?

Reply:
Yes, as part of this change I moved the logic from runtimes to the LSP router